sdk/search/azure-search-perf/src/main/java/com/azure/search/perf/Autocomplete.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.perf;
+
+import com.azure.search.documents.models.AutocompleteItem;
+import com.azure.search.perf.core.SearchPerfStressOptions;
+import com.azure.search.perf.core.ServiceTest;
+import reactor.core.publisher.Mono;
+
+/**
+ * Performs autocomplete operations.
+ */
+public class Autocomplete extends ServiceTest<SearchPerfStressOptions> {
```

Comment:
Add Test Suffix to the Test Name ? 
To have them identify as Test classes ?
Will also make it consistent with the naming pattern in other perf test projects.

Reply:
Will add `Test` suffix to all test classes.